File: pbwebbowcl/src/app/cambia-beneficiario/commons/dto/beneficiario-dto.ts
/*
 * Copyright Regione Piemonte - 2024 
 * SPDX-License-Identifier: EUPL-1.2 
*/

export class BeneficiarioDTO {
    constructor(

        public anagraficaAggiornabile : boolean,
        public codiceFiscaleSoggetto : String,
        public denominazioneBeneficiario : String,
        public idEnteGiuridico : number,
        public idProgetto : number,
        public idSoggetto : number,
        public titoloProgetto : String        
        
    ){}
}

File: pbservizit/pbservizit-jar/src/main/java/it/csi/pbandi/pbservizit/pbandisrv/dto/manager/BeneficiarioDTO.java
/*******************************************************************************
* Copyright Regione Piemonte - 2024
* SPDX-License-Identifier: EUPL-1.2
******************************************************************************/

package it.csi.pbandi.pbservizit.pbandisrv.dto.manager;


public class BeneficiarioDTO implements java.io.Serializable {

	
	static final long serialVersionUID = 1;

	private java.lang.String denominazioneBeneficiario = null;
	private Long idSoggetto;
	private java.lang.Long idTipoSoggetto = null;
	
	public java.lang.String getDenominazioneBeneficiario() {
		return denominazioneBeneficiario;
	}
	public void setDenominazioneBeneficiario(
			java.lang.String denominazioneBeneficiario) {
		this.denominazioneBeneficiario = denominazioneBeneficiario;
	}
	public Long getIdSoggetto() {
		return idSoggetto;
	}
	public void setIdSoggetto(Long idSoggetto) {
		this.idSoggetto = idSoggetto;
	}
	public java.lang.Long getIdTipoSoggetto() {
		return idTipoSoggetto;
	}
	public void setIdTipoSoggetto(java.lang.Long idTipoSoggetto) {
		this.idTipoSoggetto = idTipoSoggetto;
	}


	
}
